.buildkite/pipeline.release.yml

```diff
@@ -3,6 +3,16 @@ steps:
     prompt: "Fill out the details for release"
     if: 'build.message =~ /^release\$/i'
     fields:
+      - select: "PRE-RELEASE?"
+        key: "pre-release"
+        default: true
+        hint: "A corresponding prefix will be added to published version ('alpha' or the one specified in RELEASE_NPM_TAG)"
```

Comment:
So, I'm supposed to get `alpha` npm-tag when `pre-release == true && release-npm-tag == null`?

Reply:
by default (`RELEASE_NPM_TAG === 'null'`) `master` will return 'prepatch, preminor or premajor'.
branch will return branch name, until `RELEASE_NPM !== 'null'`

so on master we could have` 1.2.3-preminor.0` OR `1.2.3-customTag.[0..n]`
branch release could produce` 1.2.3-branchName.0 `OR `1.2.3-customTag.[0..n]`

This way it is easier to distinguish master version from branch one
@d4vidi @asafkorem